src/client.rs

```diff
@@ -1,15 +1,15 @@
 use hyper::method::Method;
 
 use hyper::header::{ContentLength, Headers};
-use log::{debug, error, trace, warn};
+use log::{debug, info, error, trace, warn, Log, Level as LogLevel, Record};
```

Comment:
Clippy suggests removing unused imports. I agree with Clippy.

Reply:
In this case, clippy is wrong.  If I remove those "unused imports", it won't compile because it's missing those symbols.

Reply:
😱

Reply:
I'll move them down to the test, but they shouldn't have resulted in warnings because they ARE used in the module (or in a child of this module).